wordle-web/src/components/SettingsPopout.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-btn icon>
+    <v-icon>mdi-cog</v-icon>
+    <v-dialog
+      width="75%"
+      height="75%"
+      v-model="dialog"
+      activator="parent"
+      transition="dialog-top-transition"
+    >
+      <v-card width="100%" height="100%">
+        <v-card-item class="justify-end">
+          <v-btn icon dark @click="dialog = false">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </v-card-item>
+        <v-card-title class="text-center">Settings</v-card-title>
+
+        <v-chip-group outlined class="justify-center ma-10 pa-10">
+          <v-card-title class="text-center">Select a color theme for the site:</v-card-title>
+
+          <v-chip size="small" @click="setDarkTheme">Dark Theme</v-chip>
+          <v-chip size="small" @click="setInverseTheme">Inverse Theme</v-chip>
```

Comment:
Why have the themes in the menu when they're also in the settings popup?

Reply:
Good point, I think we were using it to test the themes easily and forgot to remove it.